server/chat-plugins/room-events.ts

```diff
@@ -18,11 +18,28 @@ function formatEvent(event: {eventName: string, date: string, desc: string, star
 	}
 	let ret = `<tr title="${explanation}">`;
 	ret += Chat.html`<td>${event.eventName}</td>`;
+	ret += Chat.html`<td>${event.aliases ? event.aliases.join(", ") : ""}</td>`;
 	ret += `<td>${Chat.formatText(event.desc, true)}</td>`;
 	ret += Chat.html`<td><time>${event.date}</time></td></tr>`;
 	return ret;
 }
 
+function getAllAliases(room: Room) {
+	return Object.keys(room.events).map(event => room.events[event].aliases).join();
+}
```

Comment:
This... returns a string.

That's nearly definitely not what you wanted.

For one thing, if your only alias is "asdfjkl", `getAllAliases(...).includes("asdf")` is `true`, and `getAllAliases(...).includes("undefined")` is `true`, and your code doesn't actually seem to assume they wouldn't be.

Reply:
Good catch! I'm not sure how that happened but it's definitely on me. Anyway, #6805 should fix the issue.